shell/common/common_benchmarks.sh

```diff
@@ -153,6 +154,15 @@ get_options() {
         ;;
       E)
         EXPERIMENT_ID=$OPTARG
+        ;;
+      B)
+        BENCH_DATA_SIZE=$OPTARG
+        if [[ "$BENCH_DATA_SIZE" =~ ^[0-9]+(\.[0-9]+)?([MGT]B)?$ ]]; then
+          set_benchmark_data_sizes "$(get_bench_data_size_bytes "$BENCH_DATA_SIZE")"
+        else
+          logger "Invalid data size $BENCH_DATA_SIZE"
+          usage
+        fi
```

Comment:
It would be good to see the validation logic in its own function to facilitate testing, e.g.:
```
if validate_bench_data_size "$OPTARG"; then
    set_bench_data_size "$OPTARG"
else
    # report error, print usage
fi
```